### ISSUE TITLE
how to upload private keys from WebApp

### ISSUE THREAD

**Issue by piyush94** [DeployHubProject/DeployHub-Pro#244]
Is there a way to upload credential private keys from the WebApp?

**Comment by sbtaylor15** [comment]
No, right now the keys need to be accessed through the mounted volume.  Let me see where the best location to store the keys if we upload them since the container is ephemeral.

**Comment by sbtaylor15** [comment]
We can add an upload of the private key to the Credential object.  The upload would encrypt the private key and store it in the database.  The private key would be pulled from the database at deployment time, unencrypted and used for the ssh process.  

Would this work for your requirements?

**Comment by piyush94** [comment]
Yes, that would be good. Thanks.
